subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1184,6 +1197,66 @@ static int tls_opt_alpn_list_set(struct tls_context *context,
 	return 0;
 }
 
+#if defined(CONFIG_NET_SOCKETS_ENABLE_DTLS)
+static int tls_opt_dtls_handshake_timeout_get(struct tls_context *context,
+		void *optval, socklen_t *optlen, bool is_max)
+{
+	uint32_t *val = (uint32_t *)optval;
+
+	if (!optval) {
```

Comment:
`ztls_getsockopt_ctx()` already verifies `optval` and `optlen`, there's no need to repeat here.